app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RegistrationsController < Devise::RegistrationsController
+
+private
+
+  # def sign_up_params
+  #   params.require(:user).permit(:username, :email, :password, :password_confirmation)
+  # end
+  #
+  # def account_update_params
+  #   params.require(:user).permit(:username, :email, :password, :password_confirmation, :current_password)
+  # end
+
+  def after_sign_up_path_for(resource)
```

Comment:
Unused method argument - `resource`. If it's necessary, use `_` or `_resource` as an argument name to indicate that it won't be used. You can also write as `after_sign_up_path_for(*)` if you want the method to accept any arguments but don't care about them.